src/generator/AutoRest.CSharp/CodeNamerCs.cs

```diff
@@ -80,6 +80,10 @@ public override string EscapeDefaultValue(string defaultValue, IModelType type)
                 {
                     return "new " + type.Name + "()";
                 }
+                if (type is EnumType && (type as EnumType).ModelAsString)
```

Comment:
something like another code location relying on the thing being unquoted and adding its own set of quotes? sounds likely, will do :-)